clang/lib/CIR/Dialect/Transforms/LoweringPrepareItaniumCXXABI.h

```diff
@@ -21,4 +21,13 @@ class LoweringPrepareItaniumCXXABI : public cir::LoweringPrepareCXXABI {
                                cir::DynamicCastOp op) override;
   mlir::Value lowerVAArg(cir::CIRBaseBuilderTy &builder, cir::VAArgOp op,
                          const cir::CIRDataLayout &datalayout) override;
+
+  mlir::Value lowerDeleteArray(cir::CIRBaseBuilderTy &builder,
+                               cir::DeleteArrayOp op,
+                               const cir::CIRDataLayout &datalayout) override {
+    // Note: look at `CodeGenFunction::EmitCXXDeleteExpr(const CXXDeleteExpr *)`
```

Comment:
Perhaps refer to `CIRGenFunction::emitCXXDeleteExpr` instead and just add more comments there about the differences we care about detailing

Reply:
Done